firehose-setup/single-machine-deployment.md

```diff
@@ -0,0 +1,226 @@
+ls ./firehose-data/storage/one-blocks/
+
+# Inspect a specific one-block file
+firecore tools print one-block ./firehose-data/storage/one-blocks 1 --output=text
+```
+
+{% hint style="info" %}
+The output will be in protobuf text format, which is expected. This shows the raw block data as extracted by the Reader component.
+{% endhint %}
+
+{% hint style="info" %}
+One-block files contain individual block data as extracted by the Reader. Learn more about [Data Storage](../architecture/data-storage.md) patterns.
+{% endhint %}
+
+### Check Merged Blocks
+
+The Merger component combines one-block files into larger merged block files:
+
+```bash
+# List merged block files (should appear after merger processes one-blocks)
+ls ./firehose-data/storage/merged-blocks/
+
+# Inspect a merged block file (use an actual filename from the directory)
+firecore tools print merged-blocks ./firehose-data/storage/merged-blocks 100 --output=text
+```
+
+{% hint style="info" %}
+Merged blocks are optimized for efficient storage and streaming. See [Merger Component](../architecture/components/merger.md) for details.
+{% endhint %}
+
+### Check Relayer Stream
+
+The Relayer provides live block streaming:
+
+```bash
+# Stream live blocks from the relayer (in a separate terminal)
+firecore tools relayer stream localhost:10010 -o text +3
+```
+
+This command will show the last 3 blocks and then stream new blocks as they arrive.
+
+{% hint style="info" %}
+The Relayer enables real-time block streaming for live applications. Learn more about [Relayer Component](../architecture/components/relayer.md).
+{% endhint %}
+
+## Step 4: Test the gRPC API
+
+Test the Firehose gRPC API to ensure it's serving blocks correctly:
+
+```bash
+# Install grpcurl if not already available
+go install github.com/fullstorydev/grpcurl/cmd/grpcurl@latest
+
+# List available services
+grpcurl -plaintext localhost:10015 list
+
+# Get blocks from the Firehose API
+grpcurl -plaintext -d '{"start_block_num": 1, "stop_block_num": 5}' \
+  localhost:10015 sf.firehose.v2.Stream/Blocks
+```
+
+## Step 5: Test Substreams
+
+Verify that Substreams tiers are working:
+
+```bash
+# List Substreams tier1 services
+grpcurl -plaintext localhost:10016 list
+
+# List Substreams tier2 services  
+grpcurl -plaintext localhost:10017 list
+
+# Test a simple Substreams request (if you have a .spkg file)
+# substreams run -e localhost:10016 your-substream.spkg map_blocks -s 1 -t 10
+```
+
+{% hint style="info" %}
+Substreams runs on separate ports from Firehose:
+- **Substreams Tier1**: `:10016` (processing tier)
+- **Substreams Tier2**: `:10017` (caching tier)
+- **Firehose**: `:10015` (block streaming)
+{% endhint %}
+
+## Configuration Options
+
+### Storage Locations
+
+By default, all data is stored under `./firehose-data/storage/`:
+
+- **One-blocks**: `./firehose-data/storage/one-blocks/`
+- **Merged blocks**: `./firehose-data/storage/merged-blocks/`
+- **Indexes**: `./firehose-data/storage/indexes/`
```

Comment:
Remove trailing slashes.

Also document which flag controls which paths and how they are common and shared among app. You can use `docker run --rm -it ghcr.io/streamingfast/firehose-core:v1.10.1 start --help` to learn about `firecore` flags.